server.py

```
@@ -19,240 +19,323 @@
-    if not all([host, user]):
-        logger.error("Missing required environment variables:")
-        if not host: logger.error("- GERRIT_HOST not set")
-        if not user: logger.error("- GERRIT_USER not set")
+    if not token:
+        logger.error("Missing required environment variable: GITLAB_TOKEN")
         raise ValueError(
-            "Missing required environment variables: GERRIT_HOST, GERRIT_USER. "
-            "Please set these in your environment or .env file."
+            "Missing required environment variable: GITLAB_TOKEN. "
+            "Please set this in your environment or .env file."
         )
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)